admin-dev/themes/default/template/layout.tpl

```diff
@@ -31,7 +31,7 @@
 		</div>
 	</div>
 {/if}
-{if isset($error)}
+{if count($error)}
```

Comment:
I think it's better:
```suggestion
{if !empty($error)}
```

If `$error` was not defined or it is not array, it could give error.